concepticondata/conceptlists.tsv

```diff
@@ -354,7 +354,8 @@ LSDO-2019-462	Language and Social Development Organization	2019		462	basic, area
+Gerardi-2021-414	Gerardi, Fabrício and Stanislav Reichert and Coelho Aragon, Carolina	2021		414	basic	English,Portuguese	Tupían languages	https://tuled.org/parameters	Gerardi2021a		This list was compiled for a large lexical database of Tupían languages and beyond. The study we provide as reference is not identical with the concept list which we present here, as it belongs to a later version that was consistently mapped to the Concepticon. As a result, the number of items differs from the number mentioned in the study presenting the database.	1-19	
 Anonby-2018-1500	Anonby, Erik and Asadi, Ashraf and Nikravan, Pegah	2018		1500	areal, questionnaire	English, Persian	Native languages of Iran	https://carleton.ca/iran/questionnaires/	Anonby2018		This comprehensive list is used as a field work questionnaire for ALI, the Atlas of the Languages of Iran project. It includes basic as well as areally specific terms and groups them into 12 semantic categories plus multiple subcategories. Many concepts are based on [Snider-2004-1700](:ref:Snider-2004-1700).		
 Duong-2020-100	Duong, Thu Hang and Nguyen, Thu Quynh and Nguyen, Van Loi	2020		100	basic, questionnaire	English	La Chí languages in Vietnam	https://evols.library.manoa.hawaii.edu/bitstream/10524/52466/1/JSEALS_Special_Publication_6_Anthropologyoflanguage.pdf	Duong2020		The current list was used to investigate some basic characteristics of the La Chí language spoken in Bản Díu, comparing it with La Chí varieties in Bản Máy and Bản Phùng The concepts are claimed to be generally based on the 100-item list of Swadesh (presumably, the [one that dates back to 1955](:ref:Swadesh-1955-100)). The data for the analysis come from the materials collected by Nguyễn Văn Lợi, Nguyễn Thu Quỳnh and Dương Thu Hằng during their field trips in the respective regions.	136-138	
 Wittmann-1973-200	Wittmann, Henri Gontran	1973		200	basic, areal	French	Mauritian, Haitian		Wittmann1973		This list, based on [Swadesh’s 200 item list](:ref:Swadesh-1952-200), was used for lexicostatistical analysis of the relatedness between Caribbean Creoles (represented by Haitian in the list) and Mascarene Creoles (represented by Mauritian). The chosen words of both Creole languages were compared among each other and with French with regards to their homosemantic similarity, yielding a greater relatedness of each Creole to French than to each other. Our mappings generally follow the English concepts in Swadesh; however, the French translations may diverge from them in some cases.	94-98	
+Gerardi-2021-244	Ferraz Gerardi, Fabrício and Reichert, Stanislav	2021		244	basic, areal	English	Tupi-Guarani languages	https://doi.org/10.1075/dia.18032.fer	Gerardi2021b		This 244-item list was used in the analysis of sound correspondences in the 38 Tupian languages and in the development of a new internal classification of the Tupi-Guarani branch.	30-35	
```

Comment:
There was not analysis of sound correspondences in this study. So I'd say, rather:
```
This 244 item list was the basis of a pilot study on the classification of 38 Tupían languages. It was later expanded to [414 concepts](:ref:Gerardi-2021-414) as part of the TULED database. 
```